{
    "affected": [
        {
            "package": {
                "ecosystem": "FreeBSD:ports",
                "name": "python23"
            },
            "ranges": [
                {
                    "events": [
                        {
                            "fixed": "2.3.6_1"
                        },
                        {
                            "introduced": "0"
                        }
                    ],
                    "type": "ECOSYSTEM"
                }
            ]
        },
        {
            "package": {
                "ecosystem": "FreeBSD:ports",
                "name": "python24"
            },
            "ranges": [
                {
                    "events": [
                        {
                            "fixed": "2.4.5_1"
                        },
                        {
                            "introduced": "0"
                        }
                    ],
                    "type": "ECOSYSTEM"
                }
            ]
        },
        {
            "package": {
                "ecosystem": "FreeBSD:ports",
                "name": "python25"
            },
            "ranges": [
                {
                    "events": [
                        {
                            "fixed": "2.5.2_2"
                        },
                        {
                            "introduced": "0"
                        }
                    ],
                    "type": "ECOSYSTEM"
                }
            ]
        }
    ],
    "database_specific": {
        "discovery": "2008-04-10T00:00:00Z",
        "vid": "ec41c3e2-129c-11dd-bab7-0016179b2dd5"
    },
    "details": "Justin Ferguson reports:\n\n> Integer signedness error in the zlib extension module in Python\n> 2.5.2 and earlier allows remote attackers to execute arbitrary code\n> via a negative signed integer, which triggers insufficient memory\n> allocation and a buffer overflow.",
    "id": "FreeBSD-kernel-2008-0127",
    "modified": "2008-04-28T00:00:00Z",
    "published": "2008-04-25T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://api.osv.dev/v1/vulns/CVE-2008-1721"
        },
        {
            "type": "ADVISORY",
            "url": "https://www.securityfocus.com/bid/28715/info"
        },
        {
            "type": "WEB",
            "url": "http://securityreason.com/securityalert/3802"
        },
        {
            "type": "WEB",
            "url": "http://bugs.python.org/issue2586"
        }
    ],
    "schema_version": "1.7.0",
    "summary": "python -- Integer Signedness Error in zlib Module"
}
